pkgs/development/tools/misc/cc-tool/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconofHook
```

The build fails, because of a typo here and because `pkg-config` is missing.

```suggestion
, autoreconfHook
, pkg-config
```

---

Strange, as I've tried rebuilding it. Probably did not look at the output correctly and assumed that it worked, as `./result/` was still there. Sorry for the mistake.